Packages/Application/Neos.NeosIo.Event/Resources/Private/TypoScript/NodeTypes/Event.ts2

```diff
@@ -0,0 +1,45 @@
+/**
+* NodeTypes around Events
+*/
+
+
+prototype(Neos.NeosIo.Event:Event.Document) < prototype(Neos.NeosIo:LandingPage.Document) {
+	body {
+		templatePath = 'resource://Neos.NeosIo.Event/Private/Templates/Page/Event.Landingpage.html'
+		content {
+			main = Neos.NeosIo.Event:Event
+		}
+		logo = ${q(site).property('eventLogo')}
```

Comment:
Does using q(site) really work here? The conference is no new "site" as far as I can see. IMHO we need to do q(node).closest('[instanceof ...Conference]') instead?